pkg/cmd/codespace/common.go

```diff
@@ -59,6 +61,36 @@ func (a *App) StopProgressIndicator() {
 	a.io.StopProgressIndicator()
 }
 
+func startSession(ctx context.Context, codespace *api.Codespace, a *App, debug bool, debugFile string) (*liveshare.Session, func(*error), error) {
```

Comment:
Can you add a comment above for what the return values are?

Reply:
Thanks, just added